compiler/rustc_mir_build/src/build/expr/as_place.rs

```diff
@@ -217,6 +217,10 @@ fn to_upvars_resolved_place_builder<'a, 'tcx>(
                 ty::ClosureKind::FnOnce => {}
             }
 
+            // We won't be building MIR if the closure wasn't local
```

Comment:
Can call to `tcx.hir().local_def_id_to_hir_id(closure_def_id.expect_local())` be deduplicated?

Reply:
Right didn't notice there was one burried there. Thanks for catching that

Reply:
fixed